packages/components/header/_header.scss

```diff
@@ -433,13 +503,13 @@
   }
 
   &:hover {
-    background-color: $color_nhsuk-blue;
```

Comment:
I think changing this property affects the hover state on the collapsed menu view?

I noticed that when I compared the header on master versus this branch, the menu items become transparent on hover instead of blue.

Reply:
I've only applied the white hover colour to large desktop screens. So shouldn't be item transparent on collapsed menu view now https://github.com/nhsuk/nhsuk-frontend/pull/514/commits/16d3f57985646afb68224fb8d92e245ca3dcbf28